preppal-be/routes/recipeApi.ts

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 /* eslint-disable no-magic-numbers */
 const expressRecipeApi = require("express");
-const configRecipeApi = require("../configs/secrets.ts");
+// const configRecipeApi = require("../configs/secrets.ts");
```

Comment:
I just commented some lines with unused vars just in case we needed them later. But if you're certain we won't need this one, you can suggest the change on the PR here and I'll accept it

Reply:
yea just remove it. Let's avoid having commented out code.